koku/masu/util/gcp/common.py

```diff
@@ -191,9 +200,15 @@ def match_openshift_resources_and_labels(data_frame, cluster_topology, matched_t
 
     cluster_id = cluster_topology.get("cluster_id")
     cluster_alias = cluster_topology.get("cluster_alias")
+    resource_ids = cluster_topology.get("resource_ids", [])
+    resource_id_df = data_frame.get("resource_name")
 
-    LOG.info("Matching OpenShift on GCP by labels.")
-    ocp_matched = tags.str.contains(f"kubernetes-io-cluster-{cluster_id}|kubernetes-io-cluster-{cluster_alias}")
+    if resource_id_df.any():
+        LOG.info("Matching OpenShift on GCP by resource ID.")
+        ocp_matched = resource_id_df.isin(resource_ids)
```

Comment:
Is isin look for exact matches or can it do partial string matches?

Reply:
I believe it's doing partial matching based on how I was creating my resource names